src/TestFramework/Coverage/XmlReport/JUnitTestFileDataProvider.php

```diff
@@ -118,15 +114,17 @@ private function getXPath(): DOMXPath
     /**
      * @throws CoverageDoesNotExistException
      */
-    private static function createXPath(string $jUnitPath): DOMXPath
+    private static function createXPath(string $jUnitPath): SafeDOMXPath
     {
         if (!file_exists($jUnitPath)) {
             throw CoverageDoesNotExistException::forJunit($jUnitPath);
         }
 
         $dom = new DOMDocument();
-        $dom->load($jUnitPath);
+        $success = @$dom->load($jUnitPath);
```

Comment:
what's the case when we need `@` here?

Reply:
It'll emit a warning, stopping us from testing against this and other errors.